common/app/routes/challenges/components/project/Project.jsx

```diff
@@ -35,9 +35,9 @@ export class Project extends PureComponent {
   static propTypes = {
     id: PropTypes.string,
     videoId: PropTypes.string,
-    title: PropTypes.string,
     description: PropTypes.arrayOf(PropTypes.string),
-    isCompleted: PropTypes.bool
+    isCompleted: PropTypes.bool,
+    title: PropTypes.string
```

Comment:
Don't make a change that isn't required. Since title is already there there is no need to move it down.